apps/wallet/tailwind.config.js

```diff
@@ -26,7 +26,7 @@ module.exports = {
                 7.5: '1.875rem',
                 8: '2rem',
                 15: '3.75rem',
-                'popup-height': '595px',
+                'popup-height': '600px',
```

Comment:
it would be nice if we could use `POPUP_HEIGHT` here, but Tailwind doesn't support having an ES module configuration file

Reply:
It will soon 👀